src/engine/qcommon/files.cpp

```diff
@@ -639,7 +639,19 @@ void FS_LoadAllMapMetadata()
 {
 	for (const auto& pak: FS::GetAvailableMapPaks()) {
 		try {
-			FS::PakPath::LoadPakPrefix(pak, va("meta/%s/", pak.name.substr(4).c_str()));
+			std::string pak_map_prefix("map-");
```

Comment:
May be make `const` and move out of `for` loop?

Reply:
Especially useful for debug builds, where optimizations are not applied.

Reply:
Done.